server/repository/assessment/createAssessmentCycle.ts

```diff
@@ -0,0 +1,19 @@
+import { Assessment, Cycle } from '@core/meta/assessment'
+import { BaseProtocol, DB } from '@server/db'
+import { Objects } from '@core/utils'
+
+export const createAssessmentCycle = async (
```

Comment:
this method must return read from './read' which will return the assessment with the cycles